src/ccl_mass_conversion.c

```diff
@@ -118,12 +118,12 @@ void ccl_convert_concentration(ccl_cosmology *cosmo,
   int ii,st=0;
   double d_factor = delta_old/delta_new;
   for(ii=0;ii<nc;ii++) {
-    double c_new_h;
     st=convert_concentration_single(d_factor, c_old[ii], &(c_new[ii]), c_old[ii]);
     if(st!=GSL_SUCCESS) {
       *status=CCL_ERROR_ROOT;
       ccl_cosmology_set_status_message(cosmo,
-				       "ccl_mass_conversion.c: NR solver failed to find a root\n");
+        "ccl_mass_conversion.c: ccl_convert_concentration(): "
+        "NR solver failed to find a root\n");
```

Comment:
I don't think NR solver is the right thing here?

Reply:
Humm, `convert_concentration_single` uses an NR solver to translate between mass definitions, and this checks if the function has failed (which only happens if the NR solver fails).

Reply:
ahh ok. nvm then!